src/erl/tests.rs

```diff
@@ -1817,6 +1817,64 @@ main() ->
     assert_erl!(
         r#"
 pub fn main() {
+  let x = tuple(1, 2, 3)
+  case x {
+    tuple(1, 2, 3) -> 1
```

Comment:
These tests are using a tuple pattern rather than a tuple in the guard. Could we update these to something like `y if y == tuple(1, 2, 3)` please? Thanks

Reply:
Ah, of course, my bad. Fixed!